src/main/java/core/basesyntax/db/StorageDaoImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.db;
+
+import java.util.Set;
+
+public class StorageDaoImpl implements StorageDao {
+    public StorageDaoImpl() {
+    }
+
+    @Override
+    public void put(String fruit, Integer quantity) {
+        Storage.fruits.put(fruit,quantity);
+    }
+
+    @Override
+    public Integer getQuantity(String fruit) {
+        return Storage.fruits.get(fruit);

```

Comment:
The method getQuantity could potentially return a null value if the fruit does not exist in the Storage.fruits Map. You should include a null check to avoid NullPointerException.

Reply:
ok, done